src/phoenix/server/email/password_reset.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Password Reset</title>
+</head>
+<body>
+<p>Hello {{ email }},</p>
+<p>You have requested a password reset. Please click on the link below to reset your password:</p>
+<p>
+  <a id="reset-url" href="{{ base_url }}/reset-password?token={{ token }}">Reset Password</a>
```

Comment:
Wouldn't it be more secure to send the token in a request body rather a query parameter?

Reply:
good point. i wasn't sure how to do that, so will punt on that for now